src/devices/tuya.ts

```diff
@@ -8441,6 +8441,27 @@ const definitions: Definition[] = [
             tuya.whitelabel('RoomsAI', 'RoomsAI_37022454', '1 Gang switch with backlight', ['_TZ3000_w1tcofu8']),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TS0002', manufacturerName: '_TZ3000_ywubfuvt'}],
+        model: 'TS0002_zemismart',
+        vendor: 'Tuya',
+        description: '2-Gang switch with backlight, countdown and inching',
+        extend: [
+            tuya.modernExtend.tuyaOnOff({powerOnBehavior2: true, backlightModeOffOn: true, onOffCountdown: true, endpoints: ['l1', 'l2']}),
```

Comment:
Since it's tightly coupled the on/off, could you add it as an option of the `tuyaOnOff` modern extend?

Reply:
No problem, will do it.